releasetool/data/roles/upload_rpm_signatures/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: find rpms to upload signatures for
+  find:
+    path: "{{ work_dir_rpm }}"
+    patterns: "*.rpm"
+  register: rpms_to_sign
+
+- name: upload signatures
+  command: "koji import-sig {{ item.path }}"
```

Comment:
Also accepts multiple packages.

Should we use `--write` to avoid the two step process? I notice `write-signed-rpm` has an explicit gpg id where `import-sig` doesn't so maybe that's the reason?

Reply:
The reason is I am copying your scripts ;)

Reply:
And my scripts were inherited from @dLobatog who probably also inherited a more basic version :)

At least inherit the xargs style that the current scripts have :)